internal/postprocessor/go.mod

```diff
@@ -1,6 +1,6 @@
 module cloud.google.com/go/internal/postprocessor
 
-go 1.19
+go 1.20
```

Comment:
Just checking: was this intentional?

Reply:
Yes, this happened from tidying. Same with others